lib/generators/solidus_braintree/install/install_generator.rb

```diff
@@ -21,19 +21,17 @@ def add_stylesheets
       end
 
       def add_migrations
-        run 'bin/rails railties:install:migrations FROM=solidus_braintree'
+        rake 'railties:install:migrations FROM=solidus_braintree'
       end
 
       def mount_engine
-        insert_into_file File.join('config', 'routes.rb'), after: "Rails.application.routes.draw do\n" do
-          "mount SolidusBraintree::Engine, at: '/solidus_braintree'\n"
-        end
+        route "mount SolidusBraintree::Engine, at: '/solidus_braintree'"
```

Comment:
Nit: Should the commit message say `Use rake and route` instead of `Use rake and mount`?

Reply:
Yep, you're right. I'll update the commit message :+1: